ui/ts/pages/nodes.ts

```diff
@@ -333,24 +342,27 @@ module AdminViews {
               Metrics.Select.Avg(_sysMetric("cpu.sys.percent"))
                 .title("CPU Sys %")
             ).format(d3.format(".2%")).title("CPU Usage").stacked(true)
+              .tooltip("The percentage of CPU used by CockroachDB (User %) and system-level operations (Sys %) across all nodes.")
```

Comment:
Also, instead of always writing "across all nodes", let's define the base tooltip messages as constants somewhere and pull them in for the node and cluster pages and format each with the appropriate scope ("across all nodes", "on this node", etc.) and time interval ("over a 10 second period", "over a 1 hour period", etc.).